ROADMAP.md

```diff
@@ -0,0 +1,17 @@
+# Roadmap
+
+In addition to keeping up with the [issues](https://github.com/redfin/react-server/issues),
+we plan to add the following page api methods:
+
+
+`getCanonicalUrl(next: Function), optional: String | Promise(String)`
+
+* **NOT YET IMPLEMENTED**
+
+* A string (or promise thereof) that is the canonical URL for this webpage
+
+* Default: no canonical URL header added.
```

Comment:
Hmm... maybe better to track this sort of thing in issues with milestones?

Could link to milestones from this doc, but the individual work items are easier to track in the ticketing system.

Example: https://github.com/redfin/react-server/milestones/v0.3.0